Tests/mpc-kit-swiftTests/mpc_kit_swiftTests.swift

```diff
@@ -142,25 +128,26 @@ final class mpc_kit_swiftTests: XCTestCase {
         
         let deleteFactorPub = try curveSecp256k1.SecretKey(hex: newFactor).toPublic().serialize(compressed: true)
         try await coreKitInstance.deleteFactor(deleteFactorPub: deleteFactorPub, deleteFactorKey: newFactor)
-        print (keyDetails)
+        print (keyDetails ?? "")
         
     }
     
-    func testRecoveryFactor() async throws  {
+    func testMFARecoveryFactor() async throws  {
         
         let email = "testiosEmail11mfa"
         let verifier = "torus-test-health"
         let clientId = "torus-test-health"
+        // reset Account
         try await resetMPC(email: email, verifier: verifier, clientId: clientId)
         let memoryStorage = MemoryStorage()
         var coreKitInstance = MpcCoreKit( web3AuthClientId: clientId, web3AuthNetwork: Web3AuthNetwork.SAPPHIRE_DEVNET, disableHashFactor: false, localStorage: memoryStorage)
         let data = try  mockLogin2(email: email)
         let token = data
         
         
-        let keyDetails = try await coreKitInstance.loginWithJwt(verifier: verifier, verifierId: email, idToken: token)
+        _ = try await coreKitInstance.loginWithJwt(verifier: verifier, verifierId: email, idToken: token)
 
-        let hash = try Data(hex: "010203040506").sha3(varient:Variants.KECCAK256)
+//        let hash = try Data(hex: "010203040506").sha3(varient:Variants.KECCAK256)
         guard let recoveryFactor = try await coreKitInstance.enableMFA() else { throw "empty factor" };
```

Comment:
`guard` should not have to be used in tests, just the try should be sufficient.

Reply:
If the guard is still needed then the function may return nil despite the try, which really should be avoided and an error should be thrown instead.